content/pages/volunteers.md

```diff
@@ -0,0 +1,96 @@
+Title: Volunteer Opportunities
+Category: pages
+Summary: Volunteer opportunities available with WiCS undergraduate.
+
```

Comment:
files that we add can be of different types (note that I'm kinda bs-ing this but I think this is how it works)

the events show up on the home page and under the events tab

pages are like top level thing that usually show up in our nav bar (calendar, contact us, policies, etc) 

you want this to be a page because it serves the same kind of purpose as the 'contact us' page, but if you don't put `status:hidden` it shows up by default in the nav bar and we're having crowding problems there so the `status:hidden` makes it a page but a *hidden* page